### DISCUSSION TITLE
How to change caption font size of a figure in html

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#9347] [Q&A]
How to change the caption font size of a figure in html?

Reproducible example:

````qmd

---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

:::{.column-page}
![Elephant](images/elephant.jpg)
:::

````

**Comment by mcanouil** [comment]
Use your favourite browser developer/inspect mode to see what to target and use (S)CSS.

For accessibility, please add context to your post. The title is only a title. It does mean you can put only one line of code in the message itself without any messages/context.
Also, your snippet of code is not complete and is not reproducible.
Thank you for your cooperation!

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by ManuelSpinola** [comment]
Sorry about that.

El vie, 12 abr 2024 a las 0:08, Mickaël Canouil ***@***.***>)
escribió:

> Use your favourite browser developer/inspect mode to see what to target
> and use (S)CSS.
>
> For accessibility, please add context to your post. The title is only a
> title. It does you can put only one line of code in the message itself
> without any messages. Thank you for your cooperation!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/9347#discussioncomment-9091718>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBZOQ744KU6E454D6SDY45265AVCNFSM6AAAAABGDI4SVKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAOJRG4YTQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Sitio web institucional: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Sitio web personal: Sitio personal <https://mspinola-sitioweb.netlify.app>
Blog sobre Ciencia de Datos: Blog de Ciencia de Datos
<https://mspinola-ciencia-de-datos.netlify.app>

**Reply by mcanouil** [reply]
No need to apologise, editing your main post is enough.

PS: don't keep the reply when you are using mail to reply. This adds unnecessary text to your reply.
